test/ProgressOnderwijsUtils.Tests/TreeTest.cs

```diff
@@ -146,7 +146,7 @@ public void BuildWorks()
             PAssert.That(() => root_a_b.NodeValue == "b" && root_b.NodeValue == "b", "Test should select 'b' branches correctly");
         }
 
-        [Fact(Skip = "This causes OOM post-xUnit conversion; still needs to be investigated")]

```

Comment:
The actual aim of this PR was to re-enable this skipped test, because skipped tests make me nervous.  I don't believe we actually ever use this code-path heavily in progress.